src/lemlib/chassis/differential.cpp

```diff
@@ -118,13 +118,13 @@ void Differential::turnToPose(float x, float y, int timeout, bool reversed, int
  * done then is to pass the parameters to a new instance of Turn, and set the movement
  * pointer.
  */
-void Differential::turnToHeading(float heading, int timeout, int maxSpeed) {
+void Differential::turnToHeading(Angle heading, Time timeout, int maxSpeed) {
     // if a movement is already running, wait until it is done
     if (movement != nullptr) waitUntilDone();
     // convert heading to radians and standard form
-    float newHeading = M_PI_2 - degToRad(heading);
+    Angle newHeading = 90_deg - heading;
```

Comment:
```suggestion
    Angle newHeading = M_PI_2_rad - heading;
```
Change to radians for consistency